README.md

```diff
@@ -228,6 +228,42 @@ http.createServer(function (req, res) {
 
 **[Back to top](#table-of-contents)**
 
+#### Forward POST requests with a json body
+
+```ts
+const proxy = httpProxy.createProxyServer({
```

Comment:
Before the `const proxy = httpProxy.createProxyServer` create an interface `interface ExtendedIncomingMessage extends IncomingMessage {
  body?: any;
  rawBody?: Buffer | string;
}`

the reason to create this interface is to typeScript will catch errors if you try to access properties that don't exist. and to make it runtime error free